joern-cli/frontends/kotlin2cpg/src/test/scala/io/joern/kotlin2cpg/querying/LambdaTests.scala

```diff
@@ -61,8 +61,10 @@ class LambdaTests extends AnyFreeSpec with Matchers {
         |}
         |""".stripMargin)
 
-    "should contain a METHOD_REF node" in {
+    "should contain a METHOD_REF node with a non-null CODE prop set" in {
       cpg.methodRef.size shouldBe 1
+      val List(mr) = cpg.methodRef.l
+      mr.code should not be null
```

Comment:
Maybe testing the actual content of the code field here?

Reply:
Works just as well, updated